docs/documentation/concepts/items.md

```diff
@@ -28,11 +28,43 @@ The following item types are currently available (alphabetical order):
 | String         | Stores texts | String |
 | Switch         | Typically used for lights (on/off) | OnOff |
 
-Group Items can derive their own state depending on their member items.
-
-- AVG displays the average of the item states in the group.
-- OR displays an OR of the group, typically used to display whether any item in a group has been set.
-- other aggregations: AND, SUM, MIN, MAX, NAND, NOR
+## Group Items
+
+Group items collect other items into groups.
+Group items can themselves be members of other group items.
+Cyclic membership is not forbidden but strongly not recommended.
```

Comment:
Really? I thought to remember that we still had issues in such situations somewhere.

Reply:
At least the GroupItem is capable of collecting its members and not to loop indefinitely ;-) What du you suggest to state in the docs?

Reply:
How about "Cyclic memberships must be avoided."?